### DISCUSSION TITLE
Fee Elimination

### DISCUSSION THREAD

**Discussion by melpadden** [casper-network/condor-info#10] [Q&A] [answered]
@GuybrushX :

"What about gasless TXs? I remember Medha said there will be more details in a CEP so validators can decide about it but so far I didn't see a single public statement yet"

**Comment by melpadden** [comment]
An article is being prepared which discusses this in detail.

**Comment by dylanireland** [comment]
Article is being written [here](https://github.com/casper-network/condor-info/blob/fee-elimination/articles/005-fee-elimination.md).

**Comment by melpadden** [comment, marked as answer]
https://github.com/casper-network/condor-info/blob/main/articles/005-fee-elimination.md
